resources/assets/sass/_regions/_footer.scss

```diff
@@ -8,12 +8,9 @@
 [role="contentinfo"] { @include container(95%); color: #eee; padding: pxcalc(20) 0 pxcalc(50); text-align: center;
 
   .__partner {
-    span,
```

Comment:
I see there were two spans here, what was this one doing?

Reply:
the `span, img` rule was giving these items their display property, but we don't need to override the default anymore.